### DISCUSSION TITLE
--extend switch not listed in Command Details

### DISCUSSION THREAD

**Discussion by dshaw619** [gitbls/sdm#59] [Q&A] [answered]
The Command Details Wiki page (https://github.com/gitbls/sdm/wiki/Command-Details) does not document --extend as a switch to --customize.  When --customize failed for lack of space, I got the message "% If needed use --extend --xmb nnnn with --customize to create more space".

**Comment by gitbls** [comment, marked as answer]
Updated that wiki page to reflect that `--extend` can be used with `--customize`.

Thanks!
